src/loaders/TextureLoader.js

```diff
@@ -19,6 +19,12 @@ Object.assign( THREE.TextureLoader.prototype, {
 		loader.setPath( this.path );
 		loader.load( url, function ( image ) {
 
+			// JPEGs can't have an alpha channel, so memory can be saved by storing them as RGB.
+			var isJPEG = url.search(/\.(jpg|jpeg)$/) > 0;
+			if (!isJPEG) {
+				isJPEG = url.search(/^data\:image\/jpeg/) == 0;
```

Comment:
I tried to figure out if it was possible to apply this optimization to blobs as well, but couldn't find any reasonable way.